Temporary fix for our older version of Pygments in our CBE.

--- Sphinx-6.1.3/PKG-INFO
+++ Sphinx-6.1.3/PKG-INFO
@@ -42,7 +42,7 @@
 Classifier: Topic :: Text Processing :: Markup :: LaTeX
 Classifier: Topic :: Utilities
 Requires-Dist: Jinja2>=3.0
-Requires-Dist: Pygments>=2.13
+Requires-Dist: Pygments>=2.12
 Requires-Dist: docutils>=0.18,<0.20
 Requires-Dist: babel>=2.9
 Requires-Dist: alabaster>=0.7,<0.8
--- Sphinx-6.1.3/pyproject.toml
+++ Sphinx-6.1.3/pyproject.toml
@@ -56,7 +56,7 @@
 ]
 dependencies = [
     "Jinja2>=3.0",
-    "Pygments>=2.13",
+    "Pygments>=2.12",
     "docutils>=0.18,<0.20",
     "babel>=2.9",
     "alabaster>=0.7,<0.8",
